docs/guide.md

```diff
@@ -760,6 +760,11 @@ argument to your slice source.
     - `dict`: keyword arguments only;
     - Any other type is interpreted as single positional argument.
 
+You can also pass extra keyword arguments to your slice source using the 
+`slice_source_kwargs` setting. Its properties are passed as keyword arguments to 
+every slice source invocation. Keyword arguments passed as slice items with same name
+as in `slice_source_kwargs` take precedence.
```

Comment:
Could you please rephrase? I think I understand but I'm not sure, so I don't provide a suggestion.

Reply:
Fixed.